mozetl/shield/privacy_prefs.py

```diff
@@ -8,17 +8,18 @@
 from pyspark.sql.types import StringType
 
 from ..basic import convert_pings, DataFrameConfig
-from ..testpilot.utils import testpilot_etl_boilerplate
+from .utils import shield_etl_boilerplate
 
 
 SHIELD_ADDON_ID = '@shield-study-privacy'
 DATAFRAME_COLUMN_CONFIGS = [
-    ("branch", "payload/payload/branch", None, StringType()),
-    ("event", "payload/payload/event", None, StringType()),
-    ("originDomain", "payload/payload/originDomain", None, StringType()),
-    ("breakage", "payload/payload/breakage", None, StringType()),
-    ("notes", "payload/payload/notes", None, StringType()),
-    ("study", "payload/payload/study", None, StringType()),
+    ("client_id", "clientId", None, StringType()),
```

Comment:
I think a code comment is warranted here - referencing bug 1387236 and the non-standard field names